Snowflake.Data/Core/SFChunkDownloaderV2.cs

```diff
@@ -16,6 +16,7 @@
 
 namespace Snowflake.Data.Core
 {
+    [Obsolete("SFChunkDownloaderV2 is deprecated", false)]

```

Comment:
Since we don't want to remove V1 and V2 yet, maybe we could add some tests for them?
As I see there are tests commented out: TestUseV1ResultParser and TestUseV2ChunkDownloader. Please consider rewriting them, it should be easy now and will increase code coverage.

Reply:
will do, thx @sfc-gh-dstempniak good point